### DISCUSSION TITLE
[Question] How to use robot with mobile base designed in RIDGEBACK_FRANKA ?

### DISCUSSION THREAD

**Discussion by fengjiang5** [isaac-sim/IsaacLab#1125] [Q&A]
### Question

Hello, 

Thanks for your great work about robot simulation, I know [this script](https://github.com/isaac-sim/IsaacLab/blob/788a061d57ead17ff669eccce3d776fbccc59790/source/extensions/omni.isaac.lab/test/assets/check_ridgeback_franka.py) gives an examples.

**BUT**, the script is so easy, and does not contain how to control the robot(the franka arm and the mobile base).

Can you give me an example detail about this?

**Comment by fengjiang5** [comment]
Looking forward to your ealy reply!

**Comment by kellyguo11** [comment]
Hi, the script controls the robot through velocity targets and position targets. You can also check out [this page](https://isaac-sim.github.io/IsaacLab/source/overview/core-concepts/actuators.html) to learn more about actuator models.
